DataFormats/PatCandidates/src/classes_def_objects.xml

```diff
@@ -252,8 +252,8 @@
    <version ClassVersion="11" checksum="1239840459"/>
   </class>
 
-  <class name="pat::PackedCandidate" ClassVersion="16">
-   <version ClassVersion="17" checksum="1633877368"/>
+  <class name="pat::PackedCandidate" ClassVersion="17">
+    <version ClassVersion="17" checksum="1257500115"/>
```

Comment:
Hi David, I've tried rebuilding and "scram b updateclassversion" and didn't see any changes. How can I get the checksum for 18?